docs/NPCs/Xandala & Family/Alisandra Cimber.md

```diff
@@ -1,2 +1,4 @@
 Was from Chult([[The Lost City of Mezero]])
-Dead Mother of [[Xandala Cimber]] and [[Shadowmantle (aka Valindra Cimber)]], wife of [[Artus Cimber]].
+Probably dead mother of [[Xandala Cimber]] and [[Shadowmantle (aka Valindra Cimber)]], wife of [[Artus Cimber]].
+
+Weirdly, despite the pointy ears [[Alisandra Cimber]] and her daughts are not half elves.
```

Comment:
She got pointed ears from being a defender of mezro I think, and it fundamentally changed her DNA which passed it on to her daughters